### PR TITLE
Implement inspiration page intro version

### PR DESCRIPTION
**Pull request by mjpvs**
This implements a new version of the page intro component, designed for more inspirational, image centric content on mobile. The design is somewhat WIP, this version was the result of some user testing in the previous sprint but to move forward with it we want to get it up on the site on a single page (the northern lights page) to compare engagement with another, similar page using the regular general page layout with the smaller image below the title.

It's likely that, if we do retain this, it'll need to go through some iteration to ensure that it works with a wider range of images and content than specifically this one, but this initial version was put together by Aisling, Liliana and Craig as testable

Desktop retains the existing top level layout

image

Mobile has the new layout

image

### CODE REVIEW

File: src/assets/fixtures/video/dark-skies.mp4
Comment (GemmaLouise) [inline src/assets/fixtures/video/dark-skies.mp4:1]: Can this video be removed? I don't see it being used in the code but also thought we just used youtube videos

Comment (GemmaLouise) [inline src/assets/fixtures/video/dark-skies.mp4:1]: Can this video be removed? I don't see it being used in the code but also thought we just used youtube videos

File: src/components/page-intro/PageIntro.vue [hunk 1]
```diff
@@ -4,12 +4,48 @@
         :class="introClasses"
         data-test="vs-page-intro"
     >
+        <div
+            class="vs-page-intro__fs-mobile-heading-wrapper"
+            v-if="fullscreenMobile"
```
Comment (GemmaLouise) [inline src/components/page-intro/PageIntro.vue:9]: Can you add unit tests for this new logic?

**Review by GemmaLouise — COMMENTED** [review]
Can this video be removed? Can't see it being used anywhere and I thought we only had youtube videos for now?

File: src/assets/fixtures/video/dark-skies.mp4
Comment (mjpvs) [reply src/assets/fixtures/video/dark-skies.mp4:1]: Aah, yes that was the remnant of another experiment, it's gone

**Review by GemmaLouise — APPROVED** [review]

**Pull request merged**